Data Pipelines using Python.txt

```diff
@@ -46,4 +46,10 @@ Running an ingestion pipeline with Singer
 
 Getting data in to data lake
 1. With the boom of IoT, interacting with web API is becoming a common way to ingest data in to the data lake. Need to do further study in to this.
-					
\ No newline at end of file
+					
+
+REST
```

Comment:
interesting you took interest. Please read up about this in more detail. For now, move on and understand how to interact using WEB API.